src/components/toggle-button/toggle-button.stories.js

```diff
@@ -0,0 +1,172 @@
+import { html } from 'lit-html';
+import '.';
+import { action } from '@storybook/addon-actions';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { docsIntro, TYPES, CATEGORIES } from '../../../.storybook/utils.js';
+import { ICON_PLACEMENT } from './enums';
+import { argsShared, argTypesShared } from '../button/config';
+
+export default {
+  title: 'custom-elements/vl-toggle-button',
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'toggle-button',
+          intro: 'The toggle button is a button that can have an active and inactive state.',
+        }),
+      },
+    },
+  },
+  args: {
+    ...argsShared,
+    icon: undefined,
+    iconPlacement: undefined,
+    content: 'Toggle button',
+    textHidden: false,
+    disabled: false,
+    click: action('click'),
+    change: action('change'),
+  },
+  argTypes: {
+    active: {
+      name: 'active',
+      description: 'Controls the active state of the toggle button.',
+      table: {
+        type: {
+          summary: TYPES.BOOLEAN,
+        },
+        category: CATEGORIES.PROPERTIES,
+      },
+      control: { disabled: true },
+    },
+    icon: {
+      name: 'data-vl-icon',
+      type: { name: TYPES.STRING, required: false },
+      description: 'Sets the icon of the toggle button.',
+      table: {
+        type: { summary: TYPES.STRING },
+        category: CATEGORIES.ATTRIBUTES,
+      },
+    },
+    iconPlacement: {
+      name: 'data-vl-icon-placement',
+      description: 'Determines where the icon should be rendered before or after the text.',
+      table: {
+        type: { summary: TYPES.STRING },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: ICON_PLACEMENT.AFTER },
+      },
+      control: {
+        type: 'select',
+        options: [ICON_PLACEMENT.BEFORE, ICON_PLACEMENT.AFTER],
+      },
+    },
+    textHidden: {
+      name: 'data-vl-text-hidden',
+      description: 'Determines whether the toggle button text is shown.',
+      table: {
+        type: { summary: TYPES.BOOLEAN },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: 'false' },
+      },
+    },
+    change: {
+      name: 'change',
+      description: 'Custom event that is triggered when the active state of the toggle button changes.',
+      table: { category: CATEGORIES.EVENTS },
+    },
+    click: {
+      name: 'click',
+      description: 'Custom event fired on click of the toggle button.',
+      table: { category: CATEGORIES.EVENTS },
+    },
+    ...argTypesShared,
+    error: {
+      ...argTypesShared.error,
+      description:
+        'Used to emphasize the importance or consequences of an action when the toggle button is in an active state.',
+    },
+  },
+};
+
+export const Default = ({
+  icon,
```

Comment:
Ik zou hier gewoon `props` als parameter gebruiken ipv te destructuren in 13 lijnen code 😄